### PR TITLE
Mobile Skeleton 

### PR DESCRIPTION
**Pull request by lazyCode-404**
In this milestone I did the following:

- Add index.html file 
- Add style.css file 
- Configure linters
- Update README.md file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Mobile Portfolio</title>
+</head>
+<body>
+    <main>
+        <header class="header"> 
+            <a href="#">Bwalya Nonde</a>
+            <nav class="nav-menu">
+                <img src="images/union.png" alt="">
```
Comment (SajjadAhmad14) [inline index.html]: Please add a hamburger button on the right side of the header section. This button does not show on the page. Kindly fix it.
hamburger

File: index.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        <section class="introduction">
+            <div class="intro-details">
+                <h1>Bwalya Nonde</h1>
+                <h2>Software Developer</h2>
+                <p class="intro-paragraph">
+                    I can help you build a product , feature or website Look through some of 
+                    my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+                
+            </div>
```
Comment (SajjadAhmad14) [inline index.html]: Please add all the social icons which can be seen in the design. The list of icons should use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).
icons

Comment (SajjadAhmad14) [inline index.html]: Please add `Languages`, `Frameworks`, and `Skills` which can be seen in the design.
lang

File: index.html [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        </section>
```
Comment (SajjadAhmad14) [inline index.html]: Please add a button that has a link or button tag, and follow the button styling: Active, Hover, and Pressed (see design).
btn

File: index.html [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+        </header>
+        <div class="intro-photo"></div>
+        <section class="introduction">
+            <div class="intro-details">
+                <h1>Bwalya Nonde</h1>
+                <h2>Software Developer</h2>
+                <p class="intro-paragraph">
+                    I can help you build a product , feature or website Look through some of 
+                    my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
```
Comment (SajjadAhmad14) [inline index.html]: Please use the same font size for the headings and paragraph of the headline from the design.
fonts

File: style.css [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+  justify-content: space-between;
+}
+
+.nav-menu {
+  padding-right: 20px;
+  padding-top: 20px;
+}
+
+.intro-photo {
+  background-image: url("images/Ellipse18.svg");
```
Comment (SajjadAhmad14) [inline style.css]: The headline section should have a background image. Please use this image as a background for the headline section.
back

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Bwalya, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
GitHub flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by lazyCode-404** [comment]
Hello thank you so much for the review, let me make the changes.





Kind Regards,
Bwalya.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+                    <li>Github</li>
+                    <li>Codepen</li>
+                    <li>Gitlab</li>
+                    <li>Terminal</li>
+                </ul>
+              </li> 
+        </ul>
+        <hr class="hr1">
+        <ul class="social-icon">
+            <li><img src="./images/GitHub.png" alt="github-icon"></li>
```
Comment (danielufeli) [inline index.html:77]: - [ ] Nice working on your social icon lists, Like the previous reviewer said kindly link them using the `a` HTML tag

File: index.html [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+            <li><img src="./images/Linkedin.png" alt="linkedin icon"></li>
+            <li><img src="./images/Angelist.png" alt="angel-icon"></li>
+            <li><img src="./images/Twitter.png" alt="Twitter-icon"></li>
+            <li><img src="./images/media.png" alt="m-icon"></li>
+        </ul> 
+        <div class="w-arrow">
+    <ul class="wk-arr">
+        <li><img src="./images/Group55.png" alt="down-arrow icon"></li>
```
Comment (danielufeli) [inline index.html]: - [ ] Like the previous reviewer said kindly add a link to this button using the `a` HTML tag

**Review by danielufeli — CHANGES_REQUESTED** [review]
Hi @BwalyaNonde,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Nice commits messages✔️
- Descriptive PR ✔️
- Clean Design ✔️

## Required Changes ♻️
_Check the comments under the review._

 Alt text

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me with @danielufeli. in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 6 above)
Comment (lazyCode-404) [reply index.html:77]: Hey @danielufeli I am getting am error while linking the icons

Comment (danielufeli) [reply index.html:77]: > Hey @danielufeli I am getting am error while linking the icons

hi, you did not close the last `a` tag well find below 

<li><a href="#">github-icon</a></li>
            <li><a href="#">linkedin icon</a></li>
            <li><a href="#">angel-icon</a></li>
            <li><a href="#">Twitter-icon</a></li>
            <li><a href="#">m-icon   </a> </li>

**Review by vic778 — APPROVED** [review]
Hi @BwalyaNonde  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Charlie Brown

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**